app/react/components/Tip/SliderTooltip/SliderTooltip.tsx

```diff
@@ -0,0 +1,44 @@
+import Tippy from '@tippyjs/react';
+import clsx from 'clsx';
+import _ from 'lodash';
+
+import 'tippy.js/dist/tippy.css';
+
+import styles from './SliderTooltip.module.css';
+
+export interface Props {
+  value: string;
+  child: React.ReactElement;
+}
+
+export function SliderTooltip({ value, child }: Props) {
+  const id = _.uniqueId('tooltip-container-');
+
+  return (
+    <Tippy
+      className={clsx(id, styles.tooltip, '')}
+      content={messageHTML(value)}
+      placement="top"
+      showOnCreate
+      hideOnClick={false}
+      trigger="manual"
+      arrow
+      allowHTML
+    >
+      {child}
+    </Tippy>
+  );
+}
+
+function messageHTML(value: string) {
+  let message = value;
+  if (message === '0') {
+    message = 'unlimited';
+  }
+
+  return (
+    <div className={styles.tooltipContainer}>
```

Comment:
Are two nested divs required?

Reply:
Hmmmm turns out not, the inner one isn't actually doing anything, thanks!